src/component/BlockList.js

```diff
@@ -1,23 +1,61 @@
-import React from 'react';
-import { Row,Col,Image } from 'react-bootstrap';
+import React from "react";
+import "bootstrap/dist/css/bootstrap.min.css";
+import { Row, Col, Image } from "react-bootstrap";
+import Details from "./Details.js";
+const BlockList = props => {
+  const { blockList, setBlockList, addLike } = props;
+  const delBlock = item => {
+    let arr = [...blockList];
+    let idx = arr.indexOf(item);
+    if (idx !== -1) {
+      arr.splice(idx, 1);
+      setBlockList(arr);
+    }
+  };
+  const clickLike = item => {
+    addLike(item);
+    delBlock(item);
+  };
 
-const BlockList = ({ blockList }) => {
-    return (
-        <div>
-            <h1>Block List</h1>
-            <Row xs={1} md={4} lg={5} className="like-list">
-                {blockList.map(item =>
-                    <Col key={item.id}>
-                        <Image src={`https://image.tmdb.org/t/p/w500${item.poster_path}`}
-                            alt={item.title} fluid
-                        />
-                        <p>{item.title}</p>
-                        <p>{item.release_date}</p>
-                    </Col>
-                )}
-            </Row>
-        </div>
-    )
-}
+  return (
+    <div>
+      <h1>Block List</h1>
+      <Row xs={1} md={4} lg={5} className="block-list">
```

Comment:
I recommend use Container to wrap Row, which will give margin for left and right